### PR TITLE
AES XTS asm x64 MSVC

### PR DESCRIPTION
**Pull request by SparkiDev**
# Description

Use assembly code for AES-XTS with MSVC for x64.

# Testing

Jenkins

# Checklist

 - [ ] added tests
 - [ ] updated/added doxygen
 - [ ] updated appropriate READMEs
 - [ ] Updated manual and documentation

### CODE REVIEW

File: IDE/WIN10/user_settings.h [hunk 1]
```diff
@@ -117,6 +117,7 @@
         #define HAVE_FFDHE_6144
         #define HAVE_FFDHE_8192
         #define WOLFSSL_AES_OFB
+        #define WOLFSSL_AES_XTS
```
[the remark this answers is not in the document]
Comment (SparkiDev) [reply IDE/WIN10/user_settings.h]: Change made as described in diff.

File: IDE/WIN10/user_settings.h [hunk 2]
```diff
@@ -119,6 +119,9 @@
         #define WOLFSSL_AES_OFB
         #define FP_MAX_BITS 16384
     #endif /* FIPS v5 */
+    #if defined(HAVE_FIPS_VERSION) && (HAVE_FIPS_VERSION >= 7)
```
Comment (douzzer) [inline IDE/WIN10/user_settings.h:122]: This should be `HAVE_FIPS_VERSION >= 6`.

**Review by douzzer — CHANGES_REQUESTED** [review]
this has merge conflicts now.

File: IDE/WIN10/user_settings.h (hunk 2 above)
Comment (SparkiDev) [reply IDE/WIN10/user_settings.h]: fixed

**Comment by SparkiDev** [comment]
retest this please

**Review by douzzer — APPROVED** [review]

**Pull request merged**